packages/docs/src/pages/components/autocomplete/Autocomplete.vue

```diff
@@ -52,6 +52,7 @@
 <script>
     import api from './api/autocomplete'
     import variables from './variables/autocomplete'
+    import { shallowFields } from '../../../utils'
```

Comment:
@wesdevpro The following might be handy and intuitive here:
```js
import { shallowFields } from '@/utils'
```

Reply:
I didn't know that was an alias. Can we do "@" for the root directory of the docs?

Reply:
It is configured here:
https://github.com/ntohq/buefy-next/blob/723c43db202f769aa0a29ee91f3b236cea0de307/packages/docs/vite.config.js#L109

Reply:
Yup I completely missed that 👀